contrib/gofiber/fiber.v2/option.go

```diff
@@ -83,5 +83,5 @@ func WithStatusCheck(fn func(statusCode int) bool) Option {
 }
 
 func isServerError(statusCode int) bool {
```

Comment:
this function is not needed anymore

Reply:
It's used in the `defaults(cfg *config)` to set `isStatusError`, should I move `globalconfig.IsHTTPServerError(statusCode)` to inside `defaults`?

Reply:
That's what I meant yes!